### ISSUE TITLE
Spawning only in certain Areas

### ISSUE THREAD

**Issue by acmgit** [azekillDIABLO/mila#5]
Hello, azekillDiablo.
What do you find about the issue, that you add a special Option, that Mobs can only
spawn in certain Areas?
This makes it possible, to create special Forests, Caves and so on. 
Maybe a Zoo could be possible ;-).

**Comment by RodrigueLemaire** [comment]
Well. I don't really know how to do this. i could add a position ckeck
before spawning yes. but for now the developpement is frozen. this mods
needs too much change, i think I will abandon it.

**Comment by acmgit** [comment]
Yes, that's the way and sounds more difficult as it is, take a look at my Fork from mobs_redo. Bad news, that you plan to leave your Projekt. It looks like a good alternate to mobs_redo, it's never bad to have choices.

**Comment by RodrigueLemaire** [comment]
don't be sad, i will abandon it temporarly :D
